### ISSUE TITLE
Copy image to new region

### ISSUE THREAD

**Issue by BenyaminFamili** [Azure/azure-cli#11281]
## Describe the bug

**Command Name**
`az snapshot create`

**Errors:**
```
Resource imagevm_OsDisk_1_161aeed85886415e8dc2132e97b0c93a is not found.
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az snapshot create --name {} --resource-group {} --source {} --output {} --tags {}`

## Environment Summary
```
Windows-10-10.0.18362-SP0
Python 3.6.6
Shell: cmd.exe

azure-cli 2.0.76

Extensions:
image-copy-extension 0.2.1
interactive 0.4.3

```

**Comment by fengzhou-msft** [comment]
@qwordy please take a look.

**Comment by qwordy** [comment]
> 
> 
> @qwordy please take a look.

No problem

**Comment by qwordy** [comment]
I have reproduced your bug. I am debugging it.

**Comment by qwordy** [comment]
Did you delete the disk? You need to keep it. A image stores the reference of disk(s).

**Comment by qwordy** [comment]
Hi bro, is your issue solved?

**Comment by qwordy** [comment]
We have released a new version 0.2.3 recently. You can install it. It fixes a bug in snapshot create. If you have any problem, you can comment in this issue or open a new one.
